### PR TITLE
Make production branch configurable in release functions

### LINKED ISSUE #56: Support "main" production branch
The release mechanism should work with production branches called either `master` or `main`. Until now it only supports `master` branches.

### PR DESCRIPTION
**Pull request by robertauer**
Resolves #56

### CODE REVIEW

File: src/com/cloudogu/ces/cesbuildlib/GitFlow.groovy [hunk 1]
```diff
@@ -70,7 +70,7 @@ class GitFlow implements Serializable {
         git.checkout(releaseVersion)
 
         // Push changes and tags
-        git.push("origin master develop ${releaseVersion}")
+        git.push("origin " + productionBranch + " develop ${releaseVersion}")
```
Comment (schnatterer) [inline src/com/cloudogu/ces/cesbuildlib/GitFlow.groovy:73]: Why don't you use GString here - `origin ${productionBranch}..."`

File: src/com/cloudogu/ces/cesbuildlib/GitFlow.groovy [hunk 2]
```diff
@@ -25,7 +25,7 @@ class GitFlow implements Serializable {
      *
      * @param releaseVersion the version that is going to be released
      */
-    void finishRelease(String releaseVersion) {
+    void finishRelease(String releaseVersion, String productionBranch="master") {
```
Comment (schnatterer) [inline src/com/cloudogu/ces/cesbuildlib/GitFlow.groovy:28]: Formatting with spaces would make it easier to read IMO - `branch = "master"`

File: src/com/cloudogu/ces/cesbuildlib/GitHub.groovy [hunk 3]
```diff
@@ -29,7 +29,7 @@ class GitHub implements Serializable {
     /**
      * Creates a release on Github and fills it with the changes provided
      */
-    void createRelease(String releaseVersion, String changes) {
+    void createRelease(String releaseVersion, String changes, String productionBranch="master") {
```
Comment (schnatterer) [inline src/com/cloudogu/ces/cesbuildlib/GitHub.groovy:32]: Formatting with spaces would make it easier to read IMO - `branch = "master"`

**Review by schnatterer — CHANGES_REQUESTED** [review]
I only reviewed the code - don't have an environment to test those changes at hand. I'm sure somebody else could do this more efficiently. Or maybe you can point me towards it.

Are there no tests involved?

**Review by Subaro — APPROVED** [review]

**Pull request merged**